geowave-test/src/main/java/mil/nga/giat/geowave/test/service/ServicesTestEnvironment.java

```diff
@@ -96,6 +103,7 @@ public static void startServices() {
 							":",
 							";"));
 					gsWebapp.setClassLoader(classLoader);
+					gsWebapp.setParentLoaderPriority(true);
```

Comment:
Ran into imageio related sealed class exceptions without this; setting this to true creates the same behaviour as tomcat.   
@spohnan should check this for our standalone version (can also be set in the WEB-INF settings file as well for jetty)